documentation/asciidoc/computers/processors/bcm2712.adoc

```diff
@@ -24,3 +24,20 @@ Headline features include:
 ** H264 1080p30 encode (from ISP) ~30–40% CPU
 
 In aggregate, the new features present in BCM2712 deliver a performance uplift of 2-3× over Raspberry Pi 4 for common CPU or I/O-intensive use cases.
+
+=== Vulnerabilities and mitigations
+
+The Cortex-A76 used on the BCM2712 has known vulnerabilites that are all mitigated in Raspberry Pi OS.
```

Comment:
"used on" -> "used in" ?
_Perhaps_ it's worth expanding "Cortex-A76" to "Cortex-A76 CPU" and "BCM2712" to "BCM2712 SoC", for clarity? :man_shrugging:

Reply:
done